__tests__/interpolate/Remap_number_test.re

```diff
@@ -0,0 +1,61 @@
+open Jest;
+
+let it = test;
+
+describe("Remap_number", () =>
```

Comment:
`remapf` is kind of like `d3-scale` for folks who are familiar. It answers the basic question:

> Given an input range and an output domain, where would a value in the input range fall in the output domain?

`normalizef` just does this with [0, 1] as the pre-configured output range.